modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -270,7 +270,7 @@ bool AudioStreamMP3::is_monophonic() const {
 }
 
 void AudioStreamMP3::get_parameter_list(List<Parameter> *r_parameters) {
-	r_parameters->push_back(Parameter(PropertyInfo(Variant::BOOL, "looping", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_CHECKABLE), Variant()));
+	r_parameters->push_back(Parameter(PropertyInfo(Variant::BOOL, "looping", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT), Variant()));
```

Comment:
All the remaining ones are default values.
```suggestion
	r_parameters->push_back(Parameter(PropertyInfo(Variant::BOOL, "looping"), Variant()));
```